### PR TITLE
Get/Set cli for entities/relationships

### PR DESCRIPTION
**Pull request by shadansari**
```
~ $ onos topo add entity 1 --type foo
~ $ onos topo get entity 1
ID     1
Type   foo
~ $ onos topo add entity 2 --type bar
~ $ onos topo add relation 3 1 2
~ $ onos topo get relation 3
ID              3
type            UNSPECIFIED
src-entity-id   1
tgt-entity-id   2
```

### CODE REVIEW

File: pkg/cli/root.go [hunk 1]
```diff
@@ -38,15 +38,19 @@ func Init() {
...
-		Use:   "topo {get,add,update,remove,watch} [args]",
+		Use:   "topo {get,add,update,remove,watch,add-entity,get-entity,add-relation,get-relation} [args]",
 		Short: "ONOS topology subsystem commands",
 	}
 
 	clilib.AddConfigFlags(cmd, defaultAddress)
 
 	cmd.AddCommand(clilib.GetConfigCommand())
 	cmd.AddCommand(getGetCommand())
+	cmd.AddCommand(getGetEntityCommand())
```
Comment (SeanCondon) [inline pkg/cli/root.go:49]: To be more like other commands maybe get-entity and get-relationship should go in to the "get" command as "entity" and "relationship"

so the usage then would be like `onos topo get entity` etc.

same for add relationship and entity below

File: pkg/cli/topo.go [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+	objects, err := readObjects(cmd, args)
+	if err != nil {
+		return err
+	}
+
+	if len(objects) != 0 {
+		switch obj := objects[0].Obj.(type) {
+		case *topo.Object_Relationship:
+			_, _ = fmt.Fprintf(writer, "ID\t%s\n", objects[0].Ref.GetID())
```
Comment (SeanCondon) [inline pkg/cli/topo.go:117]: go text templates can be better for outputting complex lines 
See example https://github.com/onosproject/onos-config/blob/92c9bb4d0b60e144350da7e09c264128bd046854/pkg/cli/models.go#L29

File: pkg/cli/root.go (hunk 1 above)
Comment (tomikazi) [reply pkg/cli/root.go:49]: Agreed.

File: pkg/cli/topo.go (hunk 2 above)
Comment (shadansari) [reply pkg/cli/topo.go:117]: thanks for the pointer. Will look into changing it. Dont understand how to use the text template when the outputting has a for loop:

  116         case *topo.Object_Relationship:
  117             _, _ = fmt.Fprintf(writer, "ID\t%s\n", objects[0].Ref.GetID())
  118             _, _ = fmt.Fprintf(writer, "type\t%s\n", obj.Relationship.GetType())
  119             for _, ref := range obj.Relationship.GetSourceRefs() {
  120                 _, _ = fmt.Fprintf(writer, "src-entity-id\t%s\n", string(ref.ID))
  121             }
  122             for _, ref := range obj.Relationship.GetTargetRefs() {
  123                 _, _ = fmt.Fprintf(writer, "tgt-entity-id\t%s\n", string(ref.ID))
  124             }

File: pkg/cli/root.go (hunk 1 above)
Comment (shadansari) [reply pkg/cli/root.go]: fixed

File: pkg/cli/topo.go (hunk 2 above)
Comment (SeanCondon) [reply pkg/cli/topo.go:117]: @shadansari sorry for the late reply - the template can be created per line too, and/or be combined together. Like here
https://github.com/onosproject/onos-config/blob/92c9bb4d0b60e144350da7e09c264128bd046854/pkg/cli/snapshot.go#L29

**Pull request merged**